### PR TITLE
Added recipe for CSB

### PR DESCRIPTION
**Pull request by tdido**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core new recipe, review would be appreciated.

### CODE REVIEW

File: recipes/csb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    - csb-embd = csb.apps.embd:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - scipy
+    - matplotlib
+    - libgcc
```
Comment (bgruening) [inline recipes/csb/meta.yaml]: libgcc is here not needed.

**Comment by bgruening** [comment]
I love the many tests :)
One small comment inline.

**Comment by bgruening** [comment]
Thanks @tdido!

**Comment by tdido** [comment]
I got an email from travis about the [build now failing for Mac](https://travis-ci.org/bioconda/bioconda-recipes/jobs/252579708), but I can't figure out why. Any help would be appreciated, @bioconda/core.

**Comment by bgruening** [comment]
@tdido it just needed a kickstart :)
Everyhing fine now. Thanks for reporting.

**Pull request merged**